projects/igniteui-angular/src/lib/grids/grid-navigation.service.ts

```diff
@@ -221,7 +221,7 @@ export class IgxGridNavigationService {
     focusTbody(event) {
         const gridRows = this.grid.verticalScrollContainer.totalItemCount ?? this.grid.dataView.length;
         if (gridRows < 1) { this.activeNode = null; return; }
```

Comment:
@ddincheva The other option here is to change: this.activeNode = null to this.activeNode = {} as IActiveNode so we won't need the additional check in the next if. Both produce the same result in the scenario under test, so Desi, please, share your thoughts on whether one is a better option than the other.

Reply:
hi @gedinakova, according to me is better to set the active node to an object rather than null, because if there are some places where is missed the check is active node is defined this will most probably cause runtime errors. I have made a fast check and in the hierarchical grid there are such scenarios and the following errors appear 
![hGrid](https://user-images.githubusercontent.com/34240583/108707581-ea917c00-7518-11eb-9a64-bf14bb15f4f8.png)
Also as far as I know in the next major version of Angular strictNullChecks will be enabled by default so I believe is better to avoid explicitly set an object to null.

Reply:
Yes, this is just another issue I am currently addressing. Will go for the second option.